<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Магазин іграшок</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/SShop.ico</normaloff>:/icons/icons/SShop.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tablePurchase">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>87</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Артикул</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Назва</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Виробник</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кількість</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ціна</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ціна з ПДВ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сума</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="printPurchaseButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Друк</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addPurchaseButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Додати</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removePurchaseButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Видалити</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="confirmPurchasesButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Підтвердити</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page3"/>
      <widget class="QWidget" name="showpage">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="addGoodButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Додати</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="editGoodButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Редагувати</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="removeGoodButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Видалити</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableGoods">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>130</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Артикул</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Назва</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Виробник</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кількість</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ціна</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ціна з ПДВ</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>Система</string>
    </property>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Дані</string>
    </property>
    <addaction name="actionProviders"/>
    <addaction name="separator"/>
    <addaction name="actionWorkers"/>
    <addaction name="actionPost"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Налаштування</string>
    </property>
    <addaction name="actionSettingServer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Довідка</string>
    </property>
    <addaction name="actionAboutProgram"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuData"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCheck"/>
   <addaction name="actionGoods"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Вийти з системи</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Вийти</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/purchase2.ico</normaloff>:/icons/icons/purchase2.ico</iconset>
   </property>
   <property name="text">
    <string>Чек</string>
   </property>
   <property name="iconText">
    <string>Чек</string>
   </property>
   <property name="toolTip">
    <string>Чек</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGoods">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/goods.ico</normaloff>:/icons/icons/goods.ico</iconset>
   </property>
   <property name="text">
    <string>Товари</string>
   </property>
  </action>
  <action name="actionWorkers">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/people.ico</normaloff>:/icons/icons/people.ico</iconset>
   </property>
   <property name="text">
    <string>Співробітники</string>
   </property>
  </action>
  <action name="actionProviders">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/provider.ico</normaloff>:/icons/icons/provider.ico</iconset>
   </property>
   <property name="text">
    <string>Постачальники</string>
   </property>
  </action>
  <action name="actionSettingServer">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/host.ico</normaloff>:/icons/icons/host.ico</iconset>
   </property>
   <property name="text">
    <string>Серверу</string>
   </property>
  </action>
  <action name="actionPost">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/employees.ico</normaloff>:/icons/icons/employees.ico</iconset>
   </property>
   <property name="text">
    <string>Посади</string>
   </property>
  </action>
  <action name="actionAboutProgram">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/info.ico</normaloff>:/icons/icons/info.ico</iconset>
   </property>
   <property name="text">
    <string>Про программу</string>
   </property>
  </action>
  <action name="actionAddPurchase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/add.ico</normaloff>:/icons/icons/add.ico</iconset>
   </property>
   <property name="text">
    <string>Додати товар</string>
   </property>
  </action>
  <action name="actionAddGood">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/add.ico</normaloff>:/icons/icons/add.ico</iconset>
   </property>
   <property name="text">
    <string>Додати товар</string>
   </property>
  </action>
  <action name="actionEditGood">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/edit.ico</normaloff>:/icons/icons/edit.ico</iconset>
   </property>
   <property name="text">
    <string>Редагувати товар</string>
   </property>
  </action>
  <action name="actionRemoveGood">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/remove.ico</normaloff>:/icons/icons/remove.ico</iconset>
   </property>
   <property name="text">
    <string>Видалити товар</string>
   </property>
  </action>
  <action name="actionUpdateGoods">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/update.ico</normaloff>:/icons/icons/update.ico</iconset>
   </property>
   <property name="text">
    <string>Оновити товар</string>
   </property>
  </action>
  <action name="actionRemovePurchase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/remove.ico</normaloff>:/icons/icons/remove.ico</iconset>
   </property>
   <property name="text">
    <string>Видалити товар</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
